.github/CODEOWNERS

```diff
@@ -0,0 +1,85 @@
+# Global Reviewer
+.github @rancher/mapps
+docs @rancher/mapps
+scripts @rancher/mapps
+CNAME @rancher/mapps
+Makefile @rancher/mapps
+README.md @rancher/mapps
+_config.yml @rancher/mapps
+configuration.yaml @rancher/mapps
+regsync.yaml @rancher/mapps
+index.html @rancher/mapps
+
+# Elemental 
+packages/elemental @rancher/elemental
```

Comment:
@lucasmlp This path does not exist in `dev-v2.7`

Reply:
Fixed